### ISSUE TITLE
Mobile Mode Controller Support

### ISSUE THREAD

**Issue by austeregrim** [Card-Forge/forge#1141]
**Is your feature request related to a problem? Please describe.**

- I'm always frustrated when ... controller support please?

**Describe the solution you'd like**

- I'd like forge mobile to support controllers. (I don't know how, just would like it.)

**Describe alternatives you've considered**

- Staring at a blank wall, imagining what it would be like when the aliens arrive, and see me sitting here staring at said blank wall.

**Additional context**

- Well, I guess I didn't give any additional information when I wrote this. I have a device similar to the steam deck, I've added forge into the steam game launcher, it can provide joystick controls (and emulated keyboard mouse via js controller). But of course, all I can do right now is touch the screen .... ew gross.
- also uh... my car dash... has android, that screen is hot in the summer and takes my hands off the wheel. If I could just use the ... wait... if my passenger can just use a controller they don't have to reach at the screen all the time.
- ok, and I have forge installed on my amazon fire tv... you guessed it controller.
- fine... I also have these, "rokid glasses", I can plug them into my android tablet, and just play laying back in a theater mode (or AR kinda)... and yeah a controller would be really nice to not touch the screen. since I can't see where I'm putting my finger anyway.
- basically, a controller could open up so many more places forge can be played.... so much more room for activities

...
- how would this be implemented... fml I dunno man. I just visit here.

**Comment by kevlahnota** [comment]
I think I can add partial support for controller in adventure mode only, on classic mode I think it needs special attention for the GUI handling. The hard part is the battle scene (forge scene) since you need to map all necessary objects to access things and highlight all those currently selected. I'm currently looking at this: https://github.com/electronstudio/sdl2gdx
